guides/common/modules/proc_setting-a-custom-message-on-the-login-page.adoc

```diff
@@ -1,9 +1,14 @@
 [id="Setting_a_Custom_Message_on_the_Login_Page_{context}"]
 = Setting a custom message on the Login page
 
+{Project} users see a message on the {ProjectWebUI} login page before they log in.
```

Comment:
I don't think you need the first sentence because the context is clear.

Reply:
Dropped, but I had to slightly reword line 5.